02-create.md

```diff
@@ -17,7 +17,7 @@ $ mkdir planets
 $ cd planets
 ~~~
 
-and tell Git to make it a [repository](reference.html#repository)&mdash;a place where
+and tell Git to make it a [repository](reference.html#repository) &ndash; a place where
```

Comment:
It's not that you *couldn't* use an em-dash there, it just seems unnecessary. A comma would suffice and be accurate. There is no need for emphasis of the appositive phrase.